dev/Dockerfile

```diff
@@ -1,10 +1,23 @@
-FROM python:3.6-stretch
+
+ARG PYTHON_VERSION=3.6-stretch
+FROM python:${PYTHON_VERSION}
+
+ARG GITHUB_ACCOUNT=Wirecloud
+ARG GITHUB_REPOSITORY=wirecloud
+ARG SOURCE_BRANCH=master
```

Comment:
This develop image should use `develop` by default 😉.

Reply:
Fixed 89c35a3